src/systemd/sd-ndisc.h

```diff
@@ -51,6 +52,7 @@ int sd_ndisc_set_callback(sd_ndisc *nd,
                           void *userdata);
 int sd_ndisc_set_index(sd_ndisc *nd, int interface_index);
 int sd_ndisc_set_mac(sd_ndisc *nd, const struct ether_addr *mac_addr);
+int sd_ndisc_set_router_solicitation_interval(sd_ndisc *nd, usec_t router_solicitation_interval);
```

Comment:
Please use `uint64_t` instead of `usec_t` in public header file. While at it, please shorten the parameter name to just ìnterval`. The context is already set by the function name.